app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
         android:roundIcon="@mipmap/ramseslogo_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".MainActivity" android:exported="true">
```

Comment:
I don't think we should need this:
![grafik](https://user-images.githubusercontent.com/25718295/191429160-7e6cefaf-d2ef-4004-82c1-bfe722cb3efe.png)